docs/source/cpp/compute.rst

```diff
@@ -310,6 +310,20 @@ output element is null.
 | less, less_equal         |            |                                             |                     |
 +--------------------------+------------+---------------------------------------------+---------------------+
 
+These functions take any number of inputs of numeric type (in which case they
+will be cast to the :ref:`common numeric type <common-numeric-type>` before
+comparison) or of temporal types. If any input is dictionary encoded it will be
+expanded for the purposes of comparison.
+
++--------------------------+------------+---------------------------------------------+---------------------+---------------------------------------+-------+
+| Function names           | Arity      | Input types                                 | Output type         | Options class                         | Notes |
++==========================+============+=============================================+=====================+=======================================+=======+
+| maximum, minimum         | Varargs    | Numeric and Temporal                        | Numeric or Temporal | :struct:`ElementWiseAggregateOptions` | \(1)  |
```

Comment:
It just came to my mind that it's gonna be very confusing to have "minimum" and "maximum" functions in addition to "min_max". Perhaps we can be less ambiguous, e.g. "scalar_min", "scalar_max" (or "horizontal_min" or "row_min"...).

Reply:
That's a good point. I'll rename the compute function. Perhaps `elementwise_min` in accordance with the options struct (though that is a bit wordy)?

Reply:
That's wordy indeed. Or at least `element_wise_min`.

Reply:
How about naming them `least` and `greatest` like they do in SQL?

Reply:
How is that less ambiguous?

Reply:
I just changed it to element_wise_min/max.